spec/entities/base_spec.rb

```diff
@@ -51,37 +59,103 @@
   end
 
   describe 'request' do
-    let(:fixture) { File.read("spec/savon/fixtures/project/does_not_exists.xml") }
-    let(:client) { mock('client') }
-    let(:response) { mock('response') }
-    let(:request) { mock('request') }
-    let(:options) { mock('options') }
+    context 'with successful responses' do
+      let(:exists_response) { {'exists' => false} }
+      let(:bases_response) { {'bases' => {'id' => 1}} }
+      let(:prefix_bases_response) { {'bases' => {'id' => 2}} }
+
+      let(:connection) { Faraday.new { |builder| builder.adapter :test do |stubs|
+          stubs.get('/bases/1/exists') { |env| [200, {}, exists_response] }
+          stubs.get('/bases/') { |env| [200, {}, bases_response] }
+          stubs.get('/prefix/bases/') { |env| [200, {}, prefix_bases_response] }
+        end
+      }}
 
-    before :each do
-      options.expects(:timeout=)
-      options.expects(:open_timeout=)
-      request.expects(:url).with('/bases/exists')
-      request.expects(:body=).with({id: 1})
-      request.expects(:options).twice.returns(options)
+      before :each do
+        subject.class.stubs(:client).returns(connection)
+        subject.class.stubs(:endpoint).returns('bases')
+      end
+
+      context 'without an id parameter' do
+        context 'without a prefix' do
+          it 'should make the request without the prefix' do
```

Comment:
Don't be binary :smile:

 At the time of the deprecation, changing everything was way beyond what we could afford, so we start to moving slowly. Every once we modify some code we take the chance to update to the new default. If you look at the commit history for tests you can see that since `should` got deprecated we've been slowly migrating to the new pattern.

Of course if some benevolent commiter wants to migrate everything it is most welcome :smile: